cmd/bee/cmd/start.go

```diff
@@ -29,10 +29,15 @@ import (
 	"github.com/ethersphere/bee/pkg/node"
 	"github.com/ethersphere/bee/pkg/resolver/multiresolver"
 	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/kardianos/service"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
+const (
+	serviceName = "BeeSvc"
```

Comment:
I'm just thinking if we'd like to choose a longer name here, just to avoid possible conflicts with other services. Maybe `SwarmBeeSvc`

Reply:
@acud or `BeeSwarmSvc` when you open Services to be at top :)
I agree to set more unique name for service itself @zbiljic

Reply:
I am totally fine with any name. The one here was copied from some other branch.

Reply:
I don't mind either. @zbiljic I leave it up to you